gateway/src/apicast/cli/environment.lua

```diff
@@ -14,10 +14,26 @@ local assert = assert
 local error = error
 local print = print
 local pairs = pairs
+local ipairs = ipairs
+local tostring = tostring
 local insert = table.insert
 local concat = table.concat
 local re = require('ngx.re')
 
+local function parse_nameservers()
```

Comment:
Shouldn't this method be exposed in resty.resolver?

Reply:
resty.resolver exposes the nameservers as tables with some metamethods. Those objects can be joined with table.join so they have to be serialized first into `ip:port`. I guess it is worth a comment in the code. Will fix it.

Reply:
Fixed.

Reply:
That helps :+1: